tests/gnrc_tcp/tests/04-receive_data.py

```diff
@@ -58,4 +62,5 @@ def testfunc(child):
 
 
 if __name__ == '__main__':
-    sys.exit(run(testfunc, timeout=3, echo=False, traceback=True))
+    sudo_guard()
+    sys.exit(run(testfunc, timeout=20, echo=False, traceback=True))
```

Comment:
Why make it `20` for _all_ expects? It's enough to keep it to the 

```
    child.expect_exact('gnrc_tcp_recv: received ' + str(data_len), timeout=20)
```

is enough.